app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        split_integer(15, 3) == [5, 5, 5]
```

Comment:
The name of the test assumes you will calculate sum of the result

Reply:
I don't understand what I need to change, the sum of all items in the list should be equal value.

Reply:
The name of the test `def test_sum_of_the_parts_should_be_equal_to_value() -> None:` assumes that we need to check that SUM of the parts should be equal to value. It suggests that in the test we must calculate sum of the resulting list and compare it vs original value.
No need to rename this test function - just change the logic of a test slightly